app/main.ipynb

```diff
@@ -299,18 +392,29 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "result = A[::2]\n",
+    "print(result)"
```

Comment:
The selection of specific rows using slicing with a step is correct and matches the task requirement.